pkg/grafana/library-panel-handler.go

```diff
@@ -0,0 +1,168 @@
+package grafana
+
+import (
+	"fmt"
+	"path/filepath"
+
+	grafana "github.com/grafana/grafana-api-golang-client"
+	"github.com/grafana/grizzly/pkg/grizzly"
+	"github.com/grafana/tanka/pkg/kubernetes/manifest"
+)
+
+// LibraryPanelHandler is a Grizzly Handler for Grafana library panel
+type LibraryPanelHandler struct {
+	Provider Provider
+}
+
```

Comment:
nit: a type assertion here would be nice to assert that `LibraryPanelHandler` conforms to `grizzly.Handler`.